hack/build-crd-schema-checker.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euxo pipefail
+
+if [ -f "$INSTALL_DIR/crd-schema-checker" ]; then
+    exit 0
+fi
+
+mkdir -p "$INSTALL_DIR/git-tmp"
+git clone https://github.com/openshift/crd-schema-checker.git \
```

Comment:
nit: to speed up that you can limit by the max tree depth argunment

Reply:
on my machine with
*  --depth=1 it took 1.9 sec
*  --depth=1 --filter=tree:0 it took 3.8 sec
* without any filtering or limiting it took 1.4 sec
to clone the repo so I think it does not make sense to apply these for this repo. (The repo is probably small enough not to benefit from the extra steps)

Reply:
@gibizer thanks for checking! Agree this options are reserved for large repos and monorepos especially with big files that are changed at once (eg. some generated big text files that git track as a replacement). Filter can make additional request that will slow down whole cloning process. But interesting is that --depth=1 was slower at least with branch shoudn't be, but maybe it was some network hickup